_docs/performance-tuning/010-performance-tuning-introduction.md

```diff
@@ -3,9 +3,9 @@ title: "Performance Tuning Introduction"
 date:  
 parent: "Performance Tuning"
 ---
-You can apply performance tuning measures to improve how efficiently Drill queries data. To significantly improve performance in Drill, you must have knowledge about the underlying data and data sources, as well as familiarity with how Drill executes queries.
+You can change system options in Drill to improve the query performance. Before you improve performance in Drill, you must choose a layout of the data and the choose an appropriate file format specific to your use case. For example, for an analytic workload operating on historical time series data, then choosing Parquet as the file format and a partitioning scheme that uses time as a partitionining dimension would be a recommended approach. In the case you are directly querying data data sources, you need to have an understanding of the data source itself. Some familiarity with how Drill executes queries can also help. 
 
-You can analyze query plans and profiles to identify the source of performance issues in Drill. Once you have isolated the source of an issue, you can apply the following tuning techniques to improve query performance:
+You can analyze query plans and profiles to identify performance bottlenecks in Drill. Once you identified issue, here are a couple of best practices to get you started:
```

Comment:
"Once you identified issue" --> "Once you have identified an issue"
Actually, the original wording flows better...